modules/user_accounts/php/useraccountsprovisioner.class.inc

```diff
@@ -0,0 +1,65 @@
+<?php declare(strict_types=1);
+/**
+ * This file implements a data provisioner to get all modules
+ * for the User Accounts menu page.
+ *
+ * PHP Version 7
+ *
+ * @category   Core
+ * @package    Main
+ * @subpackage Core
+ * @author     Dave MacFarlane <dave.macfarlane@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+ namespace LORIS\user_accounts;
+
+ /**
+  * This class implements a data provisioner to get all modules
+  * for the User Accounts menu page.
+  *
+  * PHP Version 7
+  *
+  * @category   Core
+  * @package    Main
+  * @subpackage Core
+  * @author     Dave MacFarlane <dave.macfarlane@mcin.ca>
+  * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+  * @link       https://www.github.com/aces/Loris/
+  */
+class UserAccountsProvisioner extends \LORIS\Data\Provisioners\DBRowProvisioner
```

Comment:
Maybe this should be UserAccountsRowProvisioner?

Reply:
@driusan @ridz1208 what are your thoughts? Are we moving away from using `Row` as qualifiers for object instances? Or should keep `Row`?